### DISCUSSION TITLE
[FEATURE REQUEST] - Multiple keyboard profiles + Multiple inputs per button

### DISCUSSION THREAD

**Discussion by Greg1996** [OpenStickCommunity/GP2040-CE#835] [Ideas]
First, I wanted to thank all those developing this amazing piece of software for their hard work. I would like to request a feature for a future version of this software that allows controllers to be used as macro pads. All this would require is 1. The ability to set multiple profiles for keyboard mapping, and 2. The ability to set one button to multiple keys (i.e. set 'up' key to Control+S for save shortcut). This would allow controllers running this firmware to double as macro pads, thus extending their use even further. Thanks.

**Comment by bsstephan** [comment]
@Greg1996 - I would check the issues and make a feature request for this, if one doesn't already exist.

**Comment by TheTrainGoes** [comment]
Hi @Greg1996 ,

Can you please break this down into two seperate issues;

1 - Profiles for keyboard mode
2 - Ability to have multiple keys assigned to one GPIO

**Comment by TheTrainGoes** [comment]
Hi everyone,

This is general notice that we are closing down the discussion section of the GP2040-CE repo.  

If you would like to discuss anything further please open an issue or join our Discord:
https://discord.com/servers/openstickcommunity-1049366310389289001
